pkg/database/database.go

```diff
@@ -34,13 +34,9 @@ type DBInfoInterface interface {
 }
 
 type DBInterface interface {
-	QueryCoreResources(ctx context.Context, kind, version, limit, continueUUID, continueDate string) ([]*unstructured.Unstructured, string, string, error)
-	QueryNamespacedCoreResources(ctx context.Context, kind, version, namespace, limit, continueUUID, continueDate string) ([]*unstructured.Unstructured, string, string, error)
-	QueryNamespacedCoreResourceByName(ctx context.Context, kind, version, namespace, name string) (*unstructured.Unstructured, error)
-
-	QueryResources(ctx context.Context, kind, group, version, limit, continueUUID, continueDate string) ([]*unstructured.Unstructured, string, string, error)
-	QueryNamespacedResources(ctx context.Context, kind, group, version, namespace, limit, continueUUID, continueDate string) ([]*unstructured.Unstructured, string, string, error)
-	QueryNamespacedResourceByName(ctx context.Context, kind, group, version, namespace, name string) (*unstructured.Unstructured, error)
+	QueryResources(ctx context.Context, kind, version, limit, continueUUID, continueDate string) ([]*unstructured.Unstructured, string, string, error)
+	QueryNamespacedResources(ctx context.Context, kind, version, namespace, limit, continueUUID, continueDate string) ([]*unstructured.Unstructured, string, string, error)
+	QueryNamespacedResourceByName(ctx context.Context, kind, version, namespace, name string) (*unstructured.Unstructured, error)
```

Comment:
I would suggest renaming the `version` parameter to `apiVersion` (the same name used in the k8s resources field to wrap either the `version` or the `group/version`)
```suggestion
	QueryResources(ctx context.Context, kind, apiVersion, limit, continueUUID, continueDate string) ([]*unstructured.Unstructured, string, string, error)
	QueryNamespacedResources(ctx context.Context, kind, apiVersion, namespace, limit, continueUUID, continueDate string) ([]*unstructured.Unstructured, string, string, error)
	QueryNamespacedResourceByName(ctx context.Context, kind, apiVersion, namespace, name string) (*unstructured.Unstructured, error)
```

Reply:
It makes sense, apiVersion is the correct term here. Thanks.